scripts/main.js

```diff
@@ -214,8 +214,75 @@ function newImageUpload(uid, {fnName = "", active = true} = {}) {
   }
 
   refreshColourDisplay(uid);
+  handleDragDropEvents(uid);
 }
 
+/**
+ * Respond to file drag/drop attempts over an image form.
+ * @param {String} uid - Identifies the image form
+ */
+function handleDragDropEvents(uid) {
+  // Counter helps manage repeatedly fired drag enter+leave events
+  var counter = 0;
```

Comment:
Encapsulated the callback attachments in a function with the `counter` var to remove it from global namespace.